### PR TITLE
[FIX] web_editor: select checklist when language is rtl

### PR DESCRIPTION
**Pull request by Goaman**
task-2734731

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web_editor/static/src/scss/web_editor.common.scss [hunk 1]
```diff
@@ -135,18 +135,20 @@ ul.o_checklist {
...
+            &::before {
                 content: "✓";
-                position: absolute;
-                left: - ($o-checklist-margin-left - $o-checklist-checkmark-width);
-                top: +1px;
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                padding-left: 1px #{"/*rtl:ignore*/"};
+                padding-top: 1px;
```
Comment (dmo-odoo) [inline addons/web_editor/static/src/scss/web_editor.common.scss:151]: Could you please check in mobile ? We used to have issues with checkboxes positioning in mobile and I fear this chang might have broken it.

Comment (Goaman) [reply addons/web_editor/static/src/scss/web_editor.common.scss:151]: I've checked and it work.

**Comment by dmo-odoo** [comment]
@Goaman Runbot looks to be red.

**Comment by Goaman** [comment]
@dmo-odoo Should be fine now.

**Comment by dmo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**